jwtmiddleware.go

```diff
@@ -201,11 +199,11 @@ func (m *JWTMiddleware) CheckJWT(w http.ResponseWriter, r *http.Request) error {
 		return fmt.Errorf(errorMsg)
 	}
 
-	validToken, err := m.Options.Validate(token)
+	validToken, err := m.Options.Validate(r.Context(), token)
 
 	if err != nil {
 		m.logf("Token is invalid")
-		m.Options.ErrorHandler(w, r, "The token isn't valid")
+		m.Options.ErrorHandler(w, r, fmt.Sprintf("The token isn't valid: %s", err.Error()))
```

Comment:
```suggestion
		m.Options.ErrorHandler(w, r, fmt.Sprintf("The token isn't valid: %v", err))
```

Can just `%v` and trust the stringer bits to work.